lit_gpt/config.py

```diff
@@ -496,7 +496,42 @@ def norm_class(self) -> Type:
         norm_eps=1e-6,
         _mlp_class="LLaMAMLP",
         intermediate_size=13824,
-    )
+    ),
+    
+    # https://huggingface.co/NousResearch/Nous-Hermes-Llama2-13b
+    dict(
+        org="NousResearch",
+        name="Nous-Hermes-Llama2-13b",
+        block_size=4096,
+        vocab_size=32032,
+        n_layer=40,
+        n_head=40,
+        n_embd=5120,
+        rotary_percentage=1.0,
+        parallel_residual=False,
+        bias=False,
+        _norm_class="RMSNorm",
+        norm_eps=1e-05,
+        _mlp_class="LLaMAMLP",
+        intermediate_size=13824,
+    ),
+    
+    dict(
```

Comment:
Great job @m0saan !
But could you put `7b` before `13b` so they are listed in ascending order?